junebug/api.py

```diff
@@ -338,5 +356,70 @@ def create_router(self, request, body):
         ))
 
     @app.route('/health', methods=['GET'])
+    @inlineCallbacks
     def health_status(self, request):
-        return response(request, 'health ok', {})
+        if self.config.rabbitmq_management_interface:
+
+            ids = yield Channel.get_all(self.redis)
+
+            def get_queues(channel_ids):
+                gets = []
+                for channel_id in channel_ids:
+                    for sub in ['inbound', 'outbound', 'event']:
+
+                        queue_name = "%s.%s" % (channel_id, sub)
+                        url = 'http://%s/api/queues/%s/%s' % (
+                            self.config.rabbitmq_management_interface,
+                            self.amqp_config['vhost'].replace('/', '%2F'),
```

Comment:
I think we want to urlencode all characters in the vhost, not just the `/`

Reply:
Ah, missed this in my cleanup.